README.md

```diff
@@ -1,7 +1,45 @@
 [![Build Status](https://travis-ci.org/dedis/onet.svg?branch=master)](https://travis-ci.org/dedis/onet)
-[![Coverage Status](https://coveralls.io/repos/github/dedis/onet/badge.svg?branch=travis_24)](https://coveralls.io/github/dedis/onet?branch=travis_24)
+[![Coverage Status](https://coveralls.io/repos/github/dedis/onet/badge.svg)](https://coveralls.io/github/dedis/onet)
 
-# onet
+# The Cothority Network Library - CoNet
 
-Onet is a network library that is used to build distributed applications on a
-large scale basis.
+The Cothority-network (CoNet) is a library for simulation and deployment of
+decentralized, distributed protocols. It offers an abstraction for tree-based
+communications between thousands of nodes and is used both in research for 
+testing out new protocols and running simulations, as well as in production to
+deploy those protocols as a service in a distributed manner. A third part of the
```

Comment:
which one is the second part and the first part ? Sounds like it describe one component of the library while the others have not been described before. Before, it presents what the library can do.
Maybe write the application of the library then the components explicitely ?